### ISSUE TITLE
Rule proposal: `prefer-string-repeat`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1774]
### Description

https://github.com/eslint/eslint/pull/15731#issuecomment-1082586113

### Fail

```js
Array.from({length: 10 + 1}).join('*')
```

```js
Array.from({length: 10}).fill('*').join('')
```

```js
Array.from({length: 10}, () => '*').join('')
```

### Pass

```js
'foo'.repeat(10)
```

**Comment by brettz9** [comment]
Maybe catching `Array(10 + 1).join('*')` too for the sake of any who don't enable the rule to prefer the length argument?

**Comment by fisker** [comment]
Sure, we may check that.

**Comment by sindresorhus** [comment]
Accepted

**Comment by sindresorhus** [comment]
Fixed by https://github.com/sindresorhus/eslint-plugin-unicorn/commit/e52c455060625db4bbbe76acf8d53193d8a95c48
